### PR TITLE
verifypr: update check and docs

### PR DESCRIPTION
**Pull request by corverroos**
Updates the `verifypr` to validate feature_set tag, also update docs and PR template.

category: misc
ticket: #381

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Note: PRs can only be merged by obol-bulldozer bot. It is author's responsibilit
   - Ends with a list of tags (some required, others optional) (`^tag: value of this tag\n`):
   - `category`: required; one of: `refactor`, `bug`, `feature`, `docs`, `release`, `tidy`, `fixbuild`.
   - `ticket`: required; URL of the Github issue just a reference, E.g. `#123` or `none`.
-  - `phase`: optional; identifies the highest release cycle phase targeted by the change; `aplha`, `beta`, `gold`
+  - `feature_set`: optional; identifies the highest rollout status targeted by the change; `aplha`, `beta`, `stable`
```
Comment (dB2510) [inline docs/contributing.md:88]:
```suggestion
  - `feature_set`: optional; identifies the highest rollout status targeted by the change; `alpha`, `beta`, `stable`
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
    - 2️⃣ PR body: Replace 'body...' with detailed description of the change.
    - 3️⃣ category: pick one, delete the rest.
    - 4️⃣ ticket: Replace #000 with link to a GitHub issue (or 'none' if PR is trivial).
+   - 5️⃣ feature_set: pick one (or delete completely if not applicable).
```
Comment (dB2510) [inline .github/pull_request_template.md:7]: what are the cases of non-applicability?
maybe updating github actions would be one.

**Review by dB2510 — APPROVED** [review]
LGTM

File: .github/pull_request_template.md (hunk 2 above)
Comment (corverroos) [reply .github/pull_request_template.md:7]: Only `category: feature` requires `feature_set` I think

Comment (dB2510) [reply .github/pull_request_template.md:7]: Cool

**Pull request merged**